intermediate_source/model_parallel_tutorial.py

```diff
@@ -23,11 +23,33 @@
 of model parallel. It is up to the readers to apply the ideas to real-world
 applications.
 
-**Recommended Reading:**
-
--  https://pytorch.org/ For installation instructions
--  :doc:`/beginner/blitz/data_parallel_tutorial` Single-Machine Data Parallel
--  :doc:`/intermediate/ddp_tutorial` Combine Distributed Data Parallel and Model Parallel
+**Comparison between DataParallel and DistributedDataParallel**
+
+You may be wondering what the differences between
+`DataParallel <https://pytorch.org/docs/stable/nn.html#torch.nn.DataParallel>`_
```

Comment:
This tutorial does not cover `DataParallel`. `DataParallel` **replicates** model to different GPUs, model parallel **splits** model to different GPUs. Say a model `m` contains 10 layers, when using `DataParallel`, each GPU will have a replica of those 10 layers. When using model parallel on two GPUs, each GPU could host 5 layers.

Reply:
Got it! 

How about we cover this difference, between model parallel and data parallel, in this tutorial, and `DataParallel` vs. DDP in the next tutorial? Will do this in the revised version.